app/controllers/pictures_controller.rb

```diff
@@ -0,0 +1,28 @@
+class PicturesController < ApplicationController
+  
+  before_action :authenticate_user!, :except => [:index]
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)